### DISCUSSION TITLE
How to update the Docker Config in UnRaid to use WYZE API keys

### DISCUSSION THREAD

**Discussion by RookieITSec** [mrlt8/docker-wyze-bridge#1170] [Show and tell]
I had struggled with the various details on how to update the config to use API keys for the last few hours, only to find it was extremely simple.  

Background - I am using the docker-wyze-bridge in Docker running in Unraid (6.12.10). 

Pre-work - create your wyze API details here - https://developer-api-console.wyze.com/#/apikey/view

In UnRaid GUI -
1. Go to edit on your docker-wyze-bridge docker container to see all the settings.  
2. Go to the bottom, click "add another path, port, variable..."
3. Add a new variable.  Enter API_ID in both the name and key fields.
4. Add your wyze KEY ID to the value section.
5. Save
6. Add another new variable.  Enter API_KEY in both the name and key fields.
7. Add your wyze API KEY to the value section.
8. Save
9. Click apply at the bottom
10. From the docker unraid "home page" - restart the docker-wyze-bridge container.
11. Done.

**Comment by prinler** [comment]
I deleted the docker, installed it again and ALL the new settings popped up with ALL the new options.
